epictrack-web/src/components/insights/Work/Tabs/Staff/Charts/workListing.tsx

```diff
@@ -169,33 +163,31 @@ const WorkList = () => {
     });
   }, [workStaffs]);
 
-  const columns = React.useMemo<MRT_ColumnDef<WorkOrWorkStaff>[]>(() => {
+  const columns = React.useMemo<MRT_ColumnDef<WorkStaffWithWork>[]>(() => {
     return [
       {
-        accessorKey: "title",
+        accessorKey: "work.title",
         header: "Name",
-        size: 300,
-        Cell: canEdit
-          ? ({ row, renderedCellValue }) => (
-              <ETGridTitle
-                to="#"
-                onClick={() => {
-                  //TODO: Implement this
-                  return;
-                }}
-                titleText={row.original.title}
-              >
-                {renderedCellValue}
-              </ETGridTitle>
-            )
-          : undefined,
+        size: 200,
+        Cell: ({ row, renderedCellValue }) => {
+          return (
+            <ETGridTitle
+              to={`/work-plan?work_id=${row.original.work.id}`}
+              titleText={row.original.work.title}
+              enableTooltip
+              tooltip={row.original.work.title}
+            >
+              {row.original.work.title}
```

Comment:
Why did we remove the {renderedCellValue}. This is used to show the search highlight on the title in the grid. It was done per request before. Could you please check it with Maria

Reply:
reverted row.original.work.title i did this originally because i was debugging why the name was not displaying on the chart